Formula/z/zizmor.rb

```diff
@@ -0,0 +1,28 @@
+class Zizmor < Formula
+  desc "CLI tool for finding security issues in GitHub Actions setups"
+  homepage "https://crates.io/crates/zizmor"
+  url "https://github.com/woodruffw/zizmor/archive/refs/tags/v0.1.1.tar.gz"
+  sha256 "d743533d553538b5392ea3bfc45b27d1b55606280babd9418f9cfe7af0273ec0"
+  license "MIT"
+
+  depends_on "rust" => :build
```

Comment:
This should fix your Linux failure:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "rust" => :build

  on_linux do
    depends_on "openssl@3"
  end
```

Reply:
Thanks!